profiles/models.py

```diff
@@ -1,3 +1,447 @@
+        ordering = ["-created_at"]
+        verbose_name = "Startup Social Media"
+        verbose_name_plural = "Startup Social Media Links"
+        unique_together = ("startup", "platform")
+
+
+class Interest(models.Model):
+    STATUS_CHOICES = [
+        ('pending', 'Pending'),
+        ('acknowledged', 'Acknowledged'),
+        ('declined', 'Declined')
+    ]
+    investor = models.ForeignKey('Investor', on_delete=models.CASCADE)
+    startup = models.ForeignKey('Startup', on_delete=models.CASCADE)
+    status = models.CharField(max_length=20, choices=STATUS_CHOICES, default='pending')
+    message = models.TextField(blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return f"Interest from {self.investor.company_name} in {self.startup.company_name} - Status: {self.status}"
+
+    class Meta:
+        db_table = "interests"
+        ordering = ["-created_at"]
+        verbose_name = "Investor Interest"
+        verbose_name_plural = "Investor Interests"
+        unique_together = ("investor", "startup")
+
+
+class SavedStartup(models.Model):
+    STATUS_CHOICES = [
+        ('watching', 'Watching'),
+        ('contacted', 'Contacted'),
+        ('negotiating', 'Negotiating'),
+        ('passed', 'Passed')
+    ]
+    investor = models.ForeignKey('Investor', on_delete=models.CASCADE)
+    startup = models.ForeignKey('Startup', on_delete=models.CASCADE)
+    status = models.CharField(max_length=20, choices=STATUS_CHOICES, default='watching')
+    notes = models.TextField(blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return f"{self.investor.company_name} saved {self.startup.company_name} ({self.status})"
+
+    class Meta:
+        db_table = "saved_startups"
+        ordering = ["-created_at"]
+        verbose_name = "Saved Startup"
+        verbose_name_plural = "Saved Startups"
+        unique_together = ("investor", "startup")
+
+
+class Investment(models.Model):
+    investor = models.ForeignKey('Investor', on_delete=models.CASCADE)
+    project = models.ForeignKey('projects.Project', on_delete=models.CASCADE, null=True)
+    amount = models.DecimalField(
+        max_digits=18,
+        decimal_places=2,
+        validators=[MinValueValidator(0)]
+    )
+    percent = models.DecimalField(
+        max_digits=5,
+        decimal_places=2,
+        blank=True,
+        null=True,
+        validators=[
+            MinValueValidator(Decimal('0.00')),
+            MaxValueValidator(Decimal('100.00'))
+        ]
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def save(self, *args, **kwargs):
+        if self.project and self.amount:
+            # Calculate the total investment in this project, excluding the current record if it already exists
+            other_investments = Investment.objects.filter(project=self.project)
```

Comment:
This query executes every time save() is called. If many Investment records exist, it could impact performance.
Move this logic to a signal or service layer for clearer separation of concerns.